k8s/kubicle/cloudinit.go

```diff
@@ -0,0 +1,106 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+const workloadTemplate = `description: "{{.Description}}"
+vm_type: qemu
+fw_type: legacy
+defaults:
+    vcpus: {{.VCPUs}}
+    mem_mb: {{.RAM}}
+cloud_init: "{{.UserDataFile}}"
+disks:
+  - source:
+       service: image
+       id: "{{.ImageUUID}}"
+    size: {{.Disk}}
+    ephemeral: true
```

Comment:
What would the use case be here for wanting to preserve these volumes?  They shouldn't contain any user data, just binaries for running k8s.  They can be recreated by re-running kubicle.